src/govuk/components/radios/radios.js

```diff
@@ -40,18 +39,27 @@ Radios.prototype.setAttributes = function ($input) {
   var inputIsChecked = $input.checked
   $input.setAttribute('aria-expanded', inputIsChecked)
 
-  var $content = this.$module.querySelector('#' + $input.getAttribute('aria-controls'))
+  var $content = document.querySelector('#' + $input.getAttribute('aria-controls'))
   if ($content) {
     $content.classList.toggle('govuk-radios__conditional--hidden', !inputIsChecked)
   }
 }
 
 Radios.prototype.handleClick = function (event) {
-  nodeListForEach(this.$inputs, function ($input) {
-    // If a radio with aria-controls, handle click
-    var isRadio = $input.getAttribute('type') === 'radio'
-    var hasAriaControls = $input.getAttribute('aria-controls')
-    if (isRadio && hasAriaControls) {
+  var $clickedInput = event.target
+  // We only want to handle clicks for radio inputs
+  if ($clickedInput.type !== 'radio') {
+    return
+  }
+  // Because checking one radio can uncheck a radio in another $module,
+  // we need to call set attributes on all radios in the same form, or document if they're not in a form.
+  //
+  // We also only want radios which have aria-controls, as they support conditional reveals.
+  var $scope = ($clickedInput.form || document)
+  var $allInputs = $scope.querySelectorAll('input[type="radio"][aria-controls]')
```

Comment:
This is probably something of an edge-case but this won't query any inputs that are outside of the form but associated with it using `form="form-id"` as they won't be children of the form.

I'd suggest instead checking the form attribute of both within the forEach, i.e. `if ($input.form === $clicked.form)`